aws/resource_aws_glue_job.go

```diff
@@ -161,6 +162,10 @@ func resourceAwsGlueJobCreate(d *schema.ResourceData, meta interface{}) error {
 		input.SecurityConfiguration = aws.String(v.(string))
 	}
 
+	if v, ok := d.GetOk("tags"); ok {
```

Comment:
It's unclear to me whether the tags need to be addressed in the read and update functions

Reply:
Tags should be handled in Create, Read, and Update functions. See also: https://github.com/terraform-providers/terraform-provider-aws/blob/master/.github/CONTRIBUTING.md#resource-tagging-code-implementation